backend/src/main/java/com/fit/nufit/meal/dto/response/MealDetailResponse.java

```diff
@@ -3,21 +3,26 @@
 import com.fit.nufit.meal.domain.MealDetail;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 @AllArgsConstructor
+@NoArgsConstructor
 @Getter
 public class MealDetailResponse {
 
     private Long id;
     private Long mealId;
     private Long foodId;
     private int foodCount;
+    private int foodAmount;
+    private int calorie;
 
     public MealDetailResponse(MealDetail mealDetail) {
         this(mealDetail.getId(),
                 mealDetail.getMeal().getId(),
                 mealDetail.getFood().getId(),
-                mealDetail.getFoodCount());
-
+                mealDetail.getFoodCount(),
+                mealDetail.getFood().getAmount(),
```

Comment:
식사 상세에 표기되는 음식의 양은 음식의 개수에 비례하여 나타나는 것 같아요. 따라서 이 부분은 음식의 기본 양과 음식의 개수를 곱하여서 전달하는 것이 좋을 것 같습니다! 🙂
```suggestion
                mealDetail.getFood().getAmount() *  mealDetail.getFoodCount(),
```

Reply:
MealDetail 에 해당 기능 추가하였습니다!